app/presenters/publishing_api/consultation_presenter.rb

```diff
@@ -40,7 +40,7 @@ def links
     def edition_links
       LinksPresenter
         .new(consultation)
-        .extract(%i[organisations parent policy_areas topics government])
+        .extract(%i[organisations parent topics government])
```

Comment:
We might want to remove all these links from the Publishing API database as well - because they won't go away unless an explicit `policy_areas: []` is passed in when doing patch links.

Reply:
Yeah, maybe keeping some of this code around would be helpful if we were going to do that from Whitehall... I'm usually a proponent of using the Publishing APIs API, but in this case I think it may be better to do the cleanup from the Publishing API itself. Do you have any thoughts on this?

Reply:
Yeah, I think in this case I'd be tempted to write some sort of data migration against the Publishing API database directly.